lib/gor_project/object/object.ex

```diff
@@ -37,6 +37,26 @@ defmodule GORproject.Object do
   """
   def get_character!(id), do: Repo.get!(Character, id)
 
+  def add_stat(hash, newStat) do
+    query =
+      from(
+        c in Character,
+        where: c.hash == ^hash,
+        select: c
+      )
+
+    # add item management
+    character = Repo.one!(query)
+    stats = Poison.decode!(character.stats)
```

Comment:
Just a minor nitpick, but shouldn't you mark `add_stat` with a `!`, then? I think the `Poison.decode!/1` will never fail, but I'm less sure about the `Repo.one!/1` and the `Poison.encode!/1` which is a few lines after. Not blocking anything I think

Reply:
You are perfectly right, I figured that out myself when managing error handling, and came up with this : 

```elixir
  def add_stat(uuid, newStat) do
    query =
      from(
        c in Character,
        where: c.uuid == ^uuid,
        select: c
      )

    # add item management
    case Repo.one(query) do
      {:ok, character} ->
        stats = Poison.decode!(character.stats)

        str =
          Map.put(stats, newStat["name"], newStat["value"])
          |> Poison.encode!()

        Character.changeset(character, %{stats: str})
        |> Repo.update()

      nil ->
        {:error, "No character found for the given uuid"}
    end
  end
```

Thank's for pointing that out tho !